src/Aspire.Hosting.Testing/api/Aspire.Hosting.Testing.cs

```diff
@@ -61,7 +65,7 @@ public static System.Threading.Tasks.Task<IDistributedApplicationTestingBuilder>
             where TEntryPoint : class { throw null; }
     }
 
-    public partial interface IDistributedApplicationTestingBuilder
+    public partial interface IDistributedApplicationTestingBuilder : IDistributedApplicationBuilder, System.IAsyncDisposable, System.IDisposable
```

Comment:
breakin change?

Reply:
hrm... it wasn't flagged by package validation, not sure why

Reply:
cc @ReubenBond

Reply:
Gotta document it.

Reply:
https://github.com/dotnet/docs-aspire/issues/new?template=04-breaking-change.yml